src/main/scala/com/datastax/sparkstress/SparkCassandraStress.scala

```diff
@@ -178,12 +178,12 @@ object SparkCassandraStress {
         println(s"\nERROR: A termination time was specified with '${config.testName} which is a Read test, this is not supported yet.\n")
       } else if (!config.distributedDataTypes.contains(config.distributedDataType)) {
         println(s"\nERROR: The distributedDataType (${config.distributedDataType}) provided is not valid. Use one of these distributed data types: "+config.distributedDataTypes.mkString(", ")+".\n")
-      } else if (!config.rddSaveMethods.contains(config.saveMethod) && !config.datasetSaveMethods.contains(config.saveMethod)) {
-        println(s"\nERROR: The saveMethod (${config.saveMethod}) provided is not valid. Use one of these save methods: rdd: "+config.rddSaveMethods.mkString(", ")+", dataset: "+config.datasetSaveMethods.mkString(", ")+".\n")
+      } else if (!config.rddSaveMethods.contains(config.saveMethod) && !config.dataframeSaveMethods.contains(config.saveMethod)) {
```

Comment:
These checks are no longer needed since we ensure based on type that a dataframe save method (and distributed datat type) must be a valid one